BoardConfig.mk

```diff
@@ -28,11 +28,9 @@ BOARD_KERNEL_PAGESIZE                       := 4096
 BOARD_KERNEL_OFFSET                         := 0x00008000
 BOARD_RAMDISK_OFFSET                        := 0x01000000
 BOARD_KERNEL_TAGS_OFFSET                    := 0x00000100
+BOARD_KERNEL_CMDLINE                        := androidboot.hardware=hawaii_ss_kyleprods
```

Comment:
I can't use this,as our kernel don't pass cmdline,it get read from bootloader. Better to hardcode boot.hardware,later after boot we can think of something different but for now leave it.